Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -80,6 +80,11 @@ class MainTabBarViewController: UITabBarController {
         onFirstAppear(self)
 
         WhatsNewHandler().whatsNewViewController?.present(on: self)
+
+        if (App.shared.intercomConfig.presentMessenger) {
```

Comment:
Maybe this is not the best place for this check?

Reply:
is it possible that intercom could be presented on top of what's new screen with current implementation?

Reply:
Yes. I hope that is what happens if both are displayed.

Reply:
Should we not launch the Messenger if there is a WhatsNew screen? Or have the WhatsNew on top of the Messenger?

Reply:
in UIKit, only one screen can be presented at the same time from the same controller. This indeed may collide with what's new, and I also think that this should be either mutually exclusive or scheduled with one after another (close intercom --> show what's new).

Reply:
I think making it mutually exclusive is fine, and we should prefer intercom over the what's new, because if the intercom flag is set, then the user intends to open the intercom, and what's new is not what is expected.